datalad/distribution/install.py

```diff
@@ -285,8 +282,8 @@ class Install(Interface):
     @staticmethod
     @datasetmethod(name='install')
     def __call__(
+            source,
             path=None,
-            source=None,
```

Comment:
so get to become install... and then we would just MV get -> install ?
and installing multiple datasets "from scratch" -- multiple install commands? (i.e. like if we had to run multiple apt-get install's )

I thought we have found a peace with API already somewhat, so not sure what problem this one is solving besides may be more streamlined reimplementation...  I guess I will let you carry it on, merge #955 meanwhile (you love conflicts anyways) so we could release 0.3.1.  Such a radical change of API anyway must not happen in a bugfix release if that one to come